console-extensions.json

```diff
@@ -453,13 +453,16 @@
     }
   },
   {
-    "type": "console.dashboards/overview/health/url",
+    "type": "console.dashboards/overview/health/prometheus",
     "properties": {
-      "name": "%plugin__kubevirt-plugin~Virtualization%",
-      "url": "apis/subresources.kubevirt.io/v1/healthz",
+      "title": "%plugin__kubevirt-plugin~OpenShift Virtualization%",
+      "name": "%plugin__kubevirt-plugin~OpenShift Virtualization%",
+      "queries": ["operator_health_metric{name=\"kubevirt-hyperconvereged\"}"],
```

Comment:
The metric name is "kubevirt_hyperconverged_operator_health_status" and no label needs to be used:
```
"queries": ["kubevirt_hyperconverged_operator_health_status"],
```

Reply:
Thanks! I took the old query we discussed by mistake. I just pushed the fix.